src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -96,6 +96,11 @@ message ConfigProto {
 
   // A list of message resource name configurations.
   repeated ResourceNameMessageConfigProto resource_name_generation = 20;
+
+  // A required field to specify the version of GAPIC config schema.
+  // It is versioned using Semantic Versioning 2.0.0 (semver) and follows the
+  // semver specification. Currently, the valid values is "1.0.0".
```

Comment:
nit: 'the valid values is "1.0.0"' -> 'the only valid value is "1.0.0"'

Reply:
Done.